src/rust/catloop/mod.rs

```diff
@@ -304,6 +315,7 @@ impl SharedCatloopLibOS {
 
         // Wait for connect operation to complete.
         match queue.do_connect(remote, &yielder).await {
+            // TODO: insert into socket id to queue descriptor table?
```

Comment:
Which scenario would be prevented if this is done? Duplicate entries, like on line 201?